language-design.md

```diff
@@ -17,6 +17,18 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
+
+1. “I think that our discipline mostly uses anecdotes to argue about programming languages. As such, it is no wonder that the arguments get heated.” [Palvus, 2012]
+    I really liked this quote from Andreas Stefik, as it reflected on my experience with programming languages. Typically people base their views/arguments of PL’s on incomplete views or stereotypes. In “C is Manly, Python is for ‘n00bs’”, Linus Torvalds is cited as saying “C++ is a horrible language… made more horribly by the fact that a lot of substandard programmers use it, to the point where it’s much much easier to generate total and utter crap with it”[Yang and Rabkin, 2015] Which is exactly what Stefik is saying in this instance word for word, which has the impact of invalidating the work of all C++ developers, because of Linus Torvalds’ anecdote.
+
+2. “API design is not a solitary activity. Show your design to as many people as you can, and take their feedback seriously. Possibilities that elude your imagination may be clear to others.”[Bloch, 2006]
+    I also really liked this quote, as it reminded me of user testing in CS124: Interaction Design and reading through Don’t Make Me Think by Steve Krug. He mentions that “After you’ve worked on a site for even a few weeks, you can’t see it freshly anymore. You know too much. The only way to find out if it really works is to test it.”[Krug, 2006]. This also connects to the idea that “ Possibilities that elude your imagination may be clear to others.”
+    It also reminds me of an instance of user testing in CS124. I had created mockups in Figma for this application, and I was happy with the layout and everything to do with it, and the feedback received was “it looks like how a clown looks and I can’t read anything”, which upon looking at the mockup, it did look like how a clown looks and I could barely read anything. This was then fixed by altering the color scheme to have more contrast/more readable and legible text. The important part to me was that I was completely blind that something that I was so happy to go ahead with was almost unusable. 
+
+3. “You can always add things later, but you can't take them away.”[Bloch, 2006]
+    This quote I think is really important. I can’t remember which feature was removed because it was so long ago, but I remember the language Elm removed huge features that people had been using in the new release of Elm and people being upset over this and jumping ship to other functional languages. I remembered that it made sense to remove bloat (and to have a smaller language with fewer warts!) but that it also made very little sense to remove features already being used by people. Hence the quote “you can’t take them away” as people already depend on those things.
```

Comment:
I imagine this would be devastating for people who had already used the new features in their programs. I feel like this would also cause people to hesitate using new features in the future in fear that they could also be removed.